src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfigFilterTest.java

```diff
@@ -0,0 +1,126 @@
+package com.amazon.opendistroforelasticsearch.security.auditlog.config;
+
+import com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditCategory;
+import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
+import com.google.common.collect.ImmutableSet;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
+import static com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditCategory.*;
```

Comment:
wildcard imports are generally considered bad practice.

Reply:
I assumed it is generally fine to have wildcard imports for constants/values. 
Removing this would add 10 other import lines.

Reply:
Seeing as your using most audit categories for the tests this is a fair point. For `import static org.junit.Assert.*;` it may be better to specify the specific imports.

Reply:
Should probably write a checkstyle rule for this